perception/autoware_ground_segmentation/src/scan_ground_filter/node.cpp

```diff
@@ -176,30 +202,41 @@ void ScanGroundFilterComponent::convertPointcloud(
   const size_t in_cloud_data_size = in_cloud->data.size();
   const size_t in_cloud_point_step = in_cloud->point_step;
 
-  size_t point_index = 0;
-  pcl::PointXYZ input_point;
-  for (size_t global_offset = 0; global_offset + in_cloud_point_step <= in_cloud_data_size;
-       global_offset += in_cloud_point_step) {
-    // Point
-    get_point_from_global_offset(in_cloud, input_point, global_offset);
+  {  // add scope for time keeper
```

Comment:
It is better to describe what is done within the scope.
(The scoped-time is named "scan" and that is proper description of this scope block.) 
The time keeper is measuring that step separately.

Reply:
I changed comment to `scope for the timekeeper to track the time spent on grouping by horizontal angle`.
Also `scan` might be too abstract, I changed the name to `horizontal_angle_grouping`.